### PR TITLE
Develop to Master KeyCloak Updates + Docker instructions

### PR DESCRIPTION
**Pull request by MitchellShiell**
Before merging:

- [x] Leo wants to test Score and Keycloak running on cumulus. He will need a namespace and provide a timeline for this by eod (Thursday, Feb 29).

Summary of Content updates on this PR:

- Configuring applications for Ego UI
- Running song and score clients with Docker
- Setting up Song and Score with Keycloak for Authorization

Further consideration:

- [Running with Keycloak content seen](https://develop--overture.netlify.app/documentation/score/docker-install/installation/) will need to be updated, specifically the commit we are running

### CODE REVIEW

File: markdown/documentation/song/installation/configurations/oauth.md [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+AUTH_SERVER_TOKENNAME=apiKey
+AUTH_SERVER_KEYCLOAK_HOST={{keycloak-host-url}}
+AUTH_SERVER_KEYCLOAK_REALM=myrealm
+AUTH_SERVER_CLIENTID={{song-client-ID}}
+AUTH_SERVER_CLIENTSECRET={{song-client-secret}}
+AUTH_SERVER_SCOPE_STUDY_PREFIX=PROGRAM.study123
+AUTH_SERVER_SCOPE_STUDY_SUFFIX=.WRITE
+AUTH_SERVER_SCOPE_SYSTEM=song.WRITE
+AUTH_SERVER_INTROSPECTIONURI={{keycloak-host-url}}/realms/{{keycloak-realm}}/apikey/check_api_key/
+SPRING_SECURITY_OAUTH2_RESOURCESERVER_JWT_PUBLIC_KEY_LOCATION={{keycloak-host-url}}/realms/{{keycloak-realm}}/protocol/openid-connect/certs
```
Comment (leoraba) [inline markdown/documentation/song/installation/configurations/oauth.md:179]: use this env variable instead `SPRING_SECURITY_OAUTH2_RESOURCESERVER_JWT_JWK_SET_URI`

Comment (MitchellShiell) [reply markdown/documentation/song/installation/configurations/oauth.md:179]: Updated pending review on https://github.com/overture-stack/website/pull/381

File: markdown/documentation/ego/installation/adding-overture-services.md [hunk 2]
```diff
@@ -0,0 +1,173 @@
+---
+title: Adding Overture Services
+---
+
+# DMS-UI
```
Comment (justincorrigible) [inline markdown/documentation/ego/installation/adding-overture-services.md:5]: Thought this was already updated to Stage?

**Review by justincorrigible — APPROVED** [review]
LGTM

**Pull request merged**